SPHINXsys/src/shared/particle_dynamics/solid_dynamics/solid_dynamics.cpp

```diff
@@ -364,12 +364,32 @@ namespace SPH
 			}
 		}
 		//=================================================================================================//
+		bool checkIfPointInBoundingBox(Vecd point, BoundingBox& bbox)
```

Comment:
This is not member function. Probably put behind the definition of bounding box?

Reply:
Three functions each for 1, 2 and 3 d case?

Reply:
okey, I will move it